css/main.css

```diff
@@ -34,6 +34,70 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type="number"] {
+	max-width: 7em;
+
+	/* Remove spinner controls from Firefox */
+	appearance: textfield;
+}
+
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type="number"]:hover,
+input[type="number"]:focus {
+	appearance: number-input;
+}
+
+select[data-provides="employee"] {
+	width: 6.5em;
+}
+
+/* Enables the first input's help text to pop
+over the lasts of this group.
+Also keeps related inputs together. */
+#input fieldset {
+	white-space: nowrap;
+	display: inline-block;
+}
+
+/* Reset fieldset style */
+fieldset {
+	border: 0;
+	padding: 0;
+	padding-top: .01em;
+	margin: 0;
+	min-width: 0;
+}
+
+input#salaire:hover ~ *,
+input#salaire:focus ~ * {
+	visibility: hidden;
+}
+input#salaire:hover + label,
+input#salaire:focus + label {
+	visibility: visible;
+}
+input#salaire:hover ~ .input-help,
+input#salaire:focus ~ .input-help {
+	visibility: visible;
+	opacity: 1;
+}
+
+input#salaire ~ .input-help {
+	visibility: hidden;
+	opacity: 0;
+	transition: opacity .1s linear;
+	display: inline-block;
+	font-size: 60%;
+	margin: 0 -22em 0 .5em;
```

Comment:
> If you have another way of doing this

With `display` instead of `visibility`, so you don't need to go against the size of the hidden element.
However, since we're in an inline flow, we need to specify a `min-width` for the parent container that's larger than both the toggled elements.
I still prefer this solution as it makes this important size more visible and discoverable (common to both elements and immediately impactful on the default, non-hovered layout), but that's definitely not flawless.

> I'll add a comment
:+1:
The only important thing is to make sure someone who modifies the toggled text remembers to update this value.

- - - - -

```css
#input fieldset {
	display: inline-block;
	position: relative;
	min-width: 20em;
	text-align: left;
}

input#salaire ~ .input-help {
	font-size: 60%;
	line-height: 1.2em;
	border-left: 1px solid #CCC;
	padding: .5em;
	text-align: left;
	vertical-align: middle;
}

input#salaire:hover ~ *,
input#salaire:focus ~ *,
input#salaire ~ .input-help {
	display: none;
}
input#salaire:hover ~ .input-help,
input#salaire:focus ~ .input-help {
	display: inline-block;
}
```